NuGetPackageVerifier.json

```diff
@@ -4,16 +4,42 @@
       "AdxVerificationCompositeRule"
     ],
     "packages": {
-      "Microsoft.VisualStudio.Web.CodeGeneration.Tools": {
-        "exclusions": {
-          "BUILD_ITEMS_FRAMEWORK": {
-            "*": "This package intentionally targets .NET Standard 1.0 even though it only carries .NET Core assemblies."
+      "dotnet-aspnet-codegenerator": {
+        "Exclusions": {
+          "NEUTRAL_RESOURCES_LANGUAGE": {
+            "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg.",
+            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg."
+          },
+          "SERVICING_ATTRIBUTE": {
+            "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg.",
+            "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg."
+          },
+          "VERSION_INFORMATIONALVERSION": {
+            "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg."
+          },
+          "WRONG_PUBLICKEYTOKEN": {
+            "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg.",
+            "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg."
+          },
+          "PACKAGE_TYPE_UNEXPECTED": {
+            "*": "Assembly built by another project but bundled in our nupkg."
+          },
+          "ASSEMBLY_INFORMATIONAL_VERSION_MISMATCH": {
+            "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg.",
+            "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg."
+          },
+          "ASSEMBLY_FILE_VERSION_MISMATCH": {
+            "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg."
+          },
+          "ASSEMBLY_VERSION_MISMATCH": {
+            "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg."
           }
-        },
-        "packageTypes": [
-          "DotnetCliTool"
```

Comment:
You can remove the PACKAGE_TYPE_UNEXPECTED excludion above by adding this back and setting this to "DotnetTool"